tests/helpers/utils.py

```diff
@@ -1,14 +1,6 @@
 from pyscipopt import Model, quicksum, SCIP_PARAMSETTING, exp, log, sqrt, sin
 from typing import List
 
-from pyscipopt.scip import is_memory_freed
-
-
-def is_optimized_mode():
```

Comment:
Why are we removing this test?

Reply:
It annoyed me having this test in utils.py, since at the time the test seemed useless. I only realized after I read your comment that just constructing the model might somehow alter the result of `is_memory_free`. Still, I would prefer to have this test in a different place.

Reply:
All for it being moved if you want, but I don't see if it actually being copied somewhere else.